consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/converter/XmlToObjectTest.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.moj.sdt.cmc.consumers.client.converter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.moj.sdt.cmc.consumers.client.BaseXmlTest;
+import uk.gov.moj.sdt.cmc.consumers.converter.XmlToObjectConverter;
+import uk.gov.moj.sdt.cmc.consumers.model.breathingspace.BreathingSpaceRequest;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class XmlToObjectTest extends BaseXmlTest {
+
+    private static final String BREATHING_SPACE = "BreathingSpace.xml";
+
+    private XmlToObjectConverter xmlToObject = new XmlToObjectConverter();
+
+    @BeforeEach
+    public void setup() {
+    }
+
+    @Test
+    void shouldConvertBreathingSpaceRequestToString() throws IOException {
+        String xmlContent = readXmlAsString(BREATHING_SPACE);
+        String request = xmlToObject.convertXmlToJson(xmlContent);
+        assertNotNull(request);
+    }
+
+    @Test
+    void shouldConvertBreathingSpaceRequest() throws IOException {
+        String xmlContent = readXmlAsString(BREATHING_SPACE);
+        BreathingSpaceRequest request = xmlToObject.convertXmlToObject(xmlContent, BreathingSpaceRequest.class);
+        assertNotNull(request);
+    }
+
+    @Test
+    void shouldConvertBreathingSpaceRequestAndConvertToJson() throws IOException {
+        String xmlContent = readXmlAsString(BREATHING_SPACE);
+        BreathingSpaceRequest request = xmlToObject.convertXmlToObject(xmlContent, BreathingSpaceRequest.class);
+        ObjectMapper objectMapper = new ObjectMapper();
+        String jsonString = objectMapper.writeValueAsString(request);
+        assertNotNull(jsonString);
```

Comment:
Might be better to compare the result with an expected version - this would give more confidence that the method was doing what it's expected to do. Could add the expected result as a file if it's too large to be defined here.

Reply:
done